seclang/directives.go

```diff
@@ -470,6 +471,28 @@ func directiveSecIgnoreRuleCompilationErrors(options *DirectiveOptions) error {
 	return nil
 }
 
+func directiveSecDataset(options *DirectiveOptions) error {
+	spl := strings.SplitN(options.Opts, " ", 2)
+	if len(spl) != 2 {
+		return errors.New("syntax error: SecDataset name `\n...\n`")
```

Comment:
The error is a bit unclear to me. What does `\n...\n` mean?

Reply:
It will be printed as:
```
ERROR: Directive SecDataset - syntax error: SecDataset name `
...
`
```